gflownet/gflownet.py

```diff
@@ -100,7 +97,6 @@ def __init__(
         self.progress = progress
```

Comment:
Do we want to move this to 'config/logger.yaml` as we discussed yesterday? And then do `self.progress = self.logger.progress`?

Reply:
Yes, we do